### PR TITLE
product card was created

### PR DESCRIPTION
**Pull request by BilokonYaroslav**
- [DEMO LINK](https://BilokonYaroslav.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://BilokonYaroslav.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Comment by BilokonYaroslav** [comment]
What is this mistake 
ексіт1
and how can I avoid it? npm test gives me no errors

**Review by etojeDenys — CHANGES_REQUESTED** [review]
please rewrite styles follow css propery order https://9elements.com/css-rule-order/

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="product-card__img"
+      >
+      <div class="product-card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+      <div class="product-card__code">Product code: 195434</div>
+      <div class="product-card__response">
```
Comment (IvanFesenko) [inline src/index.html:19]: Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements
2023-02-10_15h03_21

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,42 @@
...
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="product-card__img"
+      >
+      <div class="product-card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+      <div class="product-card__code">Product code: 195434</div>
```
Comment (IvanFesenko) [inline src/index.html:18]: Name should be as heading, code as paragraph

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,42 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="product-card__reviews">Reviews: 5</div>
+      </div>
+      <div class="product-card__price-item">
+        <div class="product-card__price-text">Price:</div>
+        <div class="product-card__pice-number">$2,199</div>
```
Comment (IvanFesenko) [inline src/index.html:31]: Do not use div for text content. Use semantic tag instead

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,42 @@
...
+      <div class="product-card__code">Product code: 195434</div>
+      <div class="product-card__response">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="product-card__reviews">Reviews: 5</div>
```
Comment (IvanFesenko) [inline src/index.html:27]: Do not use div for text content. Use semantic tag instead

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Good job/ Needs to fix few items

File: src/styles/blocks/stars.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background: url(../../images/star.svg) no-repeat center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```
Comment (etojeDenys) [inline src/styles/blocks/stars.css:15]: you can just use `.stars__star:not(:last-child) { margin-right: 4px }`

**Review by etojeDenys — APPROVED** [review]
lgtm

**Review by etojeDenys — APPROVED** [review]
👍🏻

**Pull request closed without merging**